index.d.ts

```diff
@@ -92,6 +92,22 @@ declare namespace suitest {
 		 */
 		saveScreenshot(fileName: string): TakeScreenshotChain<void>;
 
+		getPairedDevice(): null | {
+			deviceId: string,
```

Comment:
Did you took this information from docs?

Reply:
no, from pariredDeviceContext